web/src/components/forms/OpsgenieDestinationForm/OpsgenieDestinationForm.tsx

```diff
@@ -45,6 +46,7 @@ const OpsgenieDestinationForm: React.FC<OpsgenieDestinationFormProps> = ({
     outputConfig: Yup.object().shape({
       opsgenie: Yup.object().shape({
         apiKey: existing ? Yup.string() : Yup.string().required(),
+        europeanServiceRegion: Yup.boolean(),
```

Comment:
This should definitely have the `.required()` method regardless of whether we end up keeping it as a boolean or not

```suggestion
        region: Yup.string().required(),
```